pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -12340,6 +12340,1693 @@ spec:
 	}
 }
 
+func TestReconciler_PipelineTaskExplicitCombosWithResults(t *testing.T) {
```

Comment:
Hey @EmmaMunley I am trying to understand the purpose of this test. I do not see the task results consumed or validated any where 🤔

Reply:
Removed this test. I think I ended up renaming it below